nornir_nautobot/plugins/tasks/dispatcher/default.py

```diff
@@ -64,11 +66,11 @@ def get_config(task: Task, logger, obj, backup_file: str, remove_lines: list, su
         running_config = result[0].result.get("config", {}).get("running", None)
         if remove_lines:
             logger.log_debug("Removing lines from configuration based on `remove_lines` definition")
```

Comment:
Nit: Update the log_debug for the new utility.

Reply:
The input "remove_lines", has not changed, I think this can be left as is.